test/e2e/flowaggregator_test.go

```diff
@@ -15,6 +15,7 @@
 package e2e
 
 import (
+	"antrea.io/antrea/pkg/agent/openflow"
```

Comment:
reminder.

Reply:
Sure, will refine it. Since I noticed a consistent failure when running kind test with flow visibility, I doubt it is possible with my change to unify the flows on +est/+rel packets. Having added debugging info and checked the test code, it should be an issue in the test cases. So I added some logic to verify if the issue could be resolved in this change.